lib/cc/analyzer/formatters/formatter.rb

```diff
@@ -2,7 +2,8 @@ module CC
   module Analyzer
     module Formatters
       class Formatter
-        def initialize(output = $stdout)
+        def initialize(filesystem, output = $stdout)
```

Comment:
The `initialize` method in the `JSONFormatter` will need adjustment to accept `filesystem` (currently takes no args)